### DISCUSSION TITLE
glitchy fonts. ; Dolphin Glitchy Fonts

### DISCUSSION THREAD

**Discussion by iEscapedVim** [prasanthrangan/hyprdots#1018] [General]
**Arch Linux**
**HP EliteBook 840 G3**
**CPU:** i5 6300U
**GPU:** Intel HD 520

https://github.com/prasanthrangan/hyprdots/assets/32537564/91aedac4-1e49-409a-b7af-67f1462ba67c

IDK what is causing that, if it's hyprland or missing driver. I don't face any issues, like lag or something, It's just super anonying.
I play Terraria, Pumpkin Panic(wine), & L4D2 and they run fine, so I don't think it could be a driver issue but I am not sure.

**Comment by kRHYME7** [comment]
Hi, is the problem only shown on Dolphin or also on other apps?

**Comment by iEscapedVim** [comment]
Only on dolphin, as far as I know

**Comment by kRHYME7** [comment]
experienced it, I guess this was dolphin related.  I will try to test some fixes.
I changed my viewing mode to detailed (ctrl + 3) so that the glitch won't be obvious 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/1b322bc4-1a0f-4387-85c0-37f7fdd02462)

**Comment by kRHYME7** [comment]
hmmm I am using a flatpak version and this shows this glitch too, I guess this can be some dolphin issues. I'll try to run it using nix to confirm further as nix package are more Isolated
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/ea6c75c2-52df-437d-aa55-2a9f49e6a491)

Update:
 Installed on nix home manager to have isolated config and yet still exhibit those errors. I guess this was not really the repo's issue and hope we can move this issue to discussions.  Maybe someone has a fix for this.
 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/23792378-3d98-4413-af5e-36900b312c61)

**Comment by iEscapedVim** [comment]
> hmmm I am using a flatpak version and this shows this glitch too, I guess this can be some dolphin issues. I'll try to run it using nix to confirm further as nix package are more Isolated

I used Dolphin with Kde and never had this issue before... could it be a missing dependency? since you mentioned that nixOS packages are isolated so It can't be a conflict right?

**Comment by kRHYME7** [comment]
> > hmmm I am using a flatpak version and this shows this glitch too, I guess this can be some dolphin issues. I'll try to run it using nix to confirm further as nix package are more Isolated
> 
> since you mentioned that nixOS packages are isolated so It can't be a conflict right?

That's how I see it, for now, maybe we lack something. But I guess the Config is not the issue. I will try to ask on hyprland Reddit, maybe we could get something.

**Comment by prasanthrangan** [comment]
oohhh, that's strange does this happen in the default view as well?

**Comment by kRHYME7** [comment]
tried it on flatpak and Nix  home manager .. My home manager uses an isolated config.

**Comment by prasanthrangan** [comment]
hmm, so its issue with dolphin itself or qt

**Comment by kRHYME7** [comment]
Hey this i fixed now. But flatpak still have this issue


```
❯ paru -Qs dolphin
local/dolphin 24.02.0-2 (kde-applications kde-system)
    KDE File Manager

```
